values.yaml

```diff
@@ -115,8 +115,8 @@ smtp:
 newrelic:
   enabled: true
   appname: Planet 4 Wordpress
-  # license:
-  # apiKey:
+  license: ""
+  apiKey: ""
```

Comment:
Does it make sense to remove NewRelic stuff here or do you prefer we do it on a second step?

Reply:
yes I think that is fine.  I have updated it to remove some old sql stuff that shouldn't have been there.